platforms/gke/base/core/deploy.sh

```diff
@@ -35,6 +35,7 @@ else
     "gke_enterprise/fleet_membership"
     # Disable gke_enterprise/servicemesh due to b/376312292
     # "gke_enterprise/servicemesh"
+    "workloads/cluster_credentials"
     "workloads/kueue"
```

Comment:
I know that this was here already, but do we expect Kueue to be needed by every deployment of the base platform?

Reply:
It is a very minimal installation and is the recommended way to consume DWS. Any of the core Terraservices can removed. I was primarily using the deploy script for CI originally and for initial deployment.